src/docs/modules/appendix/pages/release-notes.adoc

```diff
@@ -17,13 +17,29 @@ SPDX-License-Identifier: MPL-2.0
 [[bamm-2.0.0]]
 === Release 2.0.0
 
-- Introduces Entity inheritance. The new *Abstract Entity* Meta Model Element and *extends* attribute may be used to
-define a hierarchy of Entities in an Aspect Model. The concept of extending an Entity replaces the existing concept of
-refining an Entity.
+* Introduces Entity inheritance. The new *Abstract Entity* Meta Model Element and *extends*
```

Comment:
Introduce Entity inheritance, the according Meta Model Element *Abstract Entity* and *Abstract Property* as well as the *extends* attribute to define a hierarchy of Entities. This replaces the concept of refining an entity and the `bamm:refines` attribute in order to reduce complexity and modeling efforts.